examples/Atlas/urdf/atlas_convex_hull.urdf

```diff
@@ -65,7 +65,7 @@
    <collision group="heel">
       <origin rpy="0 0 0" xyz="-0.082 0.0624435 -0.081119"/>
       <geometry>
-        <sphere radius="0"/>
+        <sphere radius="1e-7"/>
```

Comment:
This is to allow bullet collision to work well with the existing contact spheres that I have to leave in place for the controller/planner. See #790